src/Entity/Notifications/Notification.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Entity\Notifications;
+
+use App\Entity\User;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\InheritanceType;
+use Doctrine\ORM\Mapping\DiscriminatorMap;
+use Doctrine\ORM\Mapping\DiscriminatorColumn;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
```

Comment:
```suggestion
 * @ORM\Entity()
```

(on each PR I have to write this comment 🙈)